### ISSUE TITLE
DistilBertModel to sequence classification

### ISSUE THREAD

**Issue by Rane90** [huggingface/transformers#21255]
### System Info

Hi,

I'm trying to create a ```DistilBertModel ``` model for sequence classification, such that ```max_position_embeddings=1024``` (otherwise I would have used ```DistilBertForSequenceClassification``` which is defult to ```max_position_embeddings=512``` )

I define the model in the following way:

```
 configuration = DistilBertConfig(max_position_embeddings=1024)
 model = DistilBertModel(configuration)
```

When forwarding an input to the model in the following way:
```
output = model(ids, attention_mask = mask, return_dict=False)[0]
```
such that ```ids.shape = (batch_size, 1024)``` and ```mask.shape = (batch_size, 1024)``` the shape of the output is ```(batch_size, 1024, 768)``` .

My question is: What is the best practice to convert this output into a probability vector over the number of labels such the modified output shape would be ``(batch_size, num_labels)```?

I thought of a few options including flattening the current output + an additional FC layer, but I'm not sure this is the best practice.

would it be possible to add a config parameter for ```DistilBertConfig``` to automatically enable this behavior?

Thank you in advance :)
@ArthurZucker , @younesbelkada

### Information

- [ ] The official example scripts
- [X] My own modified scripts

### Tasks

- [ ] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [X] My own task or dataset (give details below)

### Reproduction

```
from transformers import DistilBertConfig, DistilBertModel

configuration = DistilBertConfig(max_position_embeddings=1024)
model = transformers.BertModel.from_pretrained('bert-base-uncased')

output = self.l1(ids, mask, return_dict=False)[0]

print(output.shape)
# (batch_size, 1024, 768)
````

### Expected behavior

My question is: What is the best practice to convert this output into a probability vector over the number of labels such the modified output shape would be ``(batch_size, num_labels)```?

**Comment by ArthurZucker** [comment]
Hey! These kind of question have more sense if you ask in the [forum](https://discuss.huggingface.co/), as it is not exactly an issue nor a bug. Nevertheless, IMO you should use `DistilBertForSequenceClassification` and just modify the `max_position_embeddings`.
